pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go

```diff
@@ -67,6 +67,10 @@ type ZookeeperClusterSpec struct {
 	// PersistentVolumeClaimSpec and VolumeReclaimPolicy can be specified in here.
 	Persistence *Persistence `json:"persistence,omitempty"`
 
+	//Ephemeral is the configuration which helps create ephemeral storage
+	//At anypoint only one of Persistence or Ephemeral should be present in the manifest
+	Ephemeral *Ephemeral `json:"ephemeral,omitempty"`
```

Comment:
Can we move this under storage,  we can provide  multiple storage options under strorage like ephemeral and persistent  storage

Reply:
I have changed the structure and added storage struct and moved persistence and ephemeral in it